src/gui-wizard-gtk/ignored_words.conf

```diff
@@ -71,3 +71,12 @@ USER=
 USERNAME=
 SHELL=/sbin/nologin
 SSH_ASKPASS=
+key=0x
+keys=0x
+keyname
+key_name
```

Comment:
Perhaps pop in ‘keyfile’ (and variations) as well?

Reply:
Good idea, added :+1: